src/api_proxy/auth/auth_token.cc

```diff
@@ -21,7 +21,7 @@
 #include "src/api_proxy/auth/grpc_internals.h"
 
 extern "C" {
-  #include "grpc/support/alloc.h"
+#include "grpc/support/alloc.h"
```

Comment:
this should not in this pr

Reply:
This was from `clang-format`. Let's fix it in this PR